spreadsheet/context.go

```diff
@@ -116,6 +117,16 @@ func (e *evalContext) GetLocked(cellRef string) bool {
 	return e.s.Cell(cellRef).getLocked()
 }
 
+// HasFormula returns if the cell contains formula.
+func (e *evalContext) HasFormula(cellRef string) bool {
+	return e.s.Cell(cellRef).HasFormula()
+}
+
+// IsBool returns if the cell boolean value.
```

Comment:
returns true if the cell is a boolean value

Reply:
fixed